packages/admin/cms-admin/src/blocks/InternalLinkBlock.tsx

```diff
@@ -57,8 +57,10 @@ export const InternalLinkBlock: BlockInterface<InternalLinkBlockData, State, Int
             variables: { id: output.targetPageId },
         });
 
+        const targetPage = data.pageTreeNode ? { ...data.pageTreeNode, scope: pageTreeScope } : undefined;
```

Comment:
@nsams we decided to not load the scope dynamically but instead patch it into the response. Reasons:
1. We're currently doing the same in the PageTreeSelectDialog: https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/pages/pageTreeSelect/PageTreeSelectDialog.tsx#L169-L175 (for new link blocks). 
2. Having a dynamically generated query results in GraphQL Codegen no longer working, having to define the types manually.

Do you agree that this is fine? Otherwise we'll have to change it in both places.

Reply:
that only works if there are no links linking to a different scope. Can we guarantee that, also for future projects?

Reply:
> Can we guarantee that, also for future projects?

No, because at same point we'll have to implement linking to other pages (/kontakt -> /contact) I suppose, but we can fix it then. 😁